rtcpeerconnection.js

```diff
@@ -617,11 +617,15 @@ module.exports = function(window, edgeVersion) {
     var params = getCommonCapabilities(transceiver.localCapabilities,
         transceiver.remoteCapabilities);
     if (send && transceiver.rtpSender) {
-      params.encodings = transceiver.sendEncodingParameters;
+      if (transceiver.sendEncodingParameters.length) {
+        params.encodings = transceiver.sendEncodingParameters;
+      }
       params.rtcp = {
-        cname: SDPUtils.localCName,
```

Comment:
this will always be set unless there is a major version of the sdp module.

Reply:
This is the same as my previous response - Changed only to make it look the same as the code handing the remote SDP. So again you are correct, probably not needed.